tfhe/src/core_crypto/algorithms/lwe_encryption.rs

```diff
@@ -1368,7 +1368,7 @@ mod test {
             );
             // Create the plaintext
             let msg: Scalar = test_tools::random_uint_between(Scalar::ZERO..Scalar::TWO.shl(2));
-            let encoded_msg = msg << 60;
+            let encoded_msg = msg << 28;
```

Comment:
```
msg << (Scalar::BITS - 4)
```
that's better for the spirit of the encoding we were kind of doing here

Reply:
Shouldn't it be `msg << (Scalar::BITS - 5)` for the padding bit ?

Reply:
strictly yes